homeassistant/components/harmony/config_flow.py

```diff
@@ -150,14 +150,11 @@ def async_get_options_flow(config_entry):
         if self._host_already_configured(validated):
```

Comment:
Do we still need this if we can guarantee a unique id for the config entry now?

Reply:
I don't think its needed anymore. retesting.....

Reply:
All good without it.

Removed

<img width="431" alt="Screen Shot 2020-03-19 at 7 59 14 PM" src="https://user-images.githubusercontent.com/663432/77127900-34760600-6a1c-11ea-9159-3738d8ac4b16.png">